app/authentication_views.py

```diff
@@ -21,49 +22,25 @@ def register():
             "Handling '/register' route,  user is authenticated, redirecting to route '/home'")
         return redirect(url_for("home"))
 
-    if request.method == "POST":
-
-        username = request.form['username']
-        email = request.form['email']
-        password = request.form['password']
-        birthday = request.form['birthday']
 
+    form = RegistrationForm(request.form)
+    # TODO find a way to give information to user that email is already used.
+    if request.method == "POST" and form.validate():
```

Comment:
shouldnt you return some specific error to user if form is not valid?

Reply:
it does, that method checks all validations that are indicated in RegisterForm class and display an error message to a user in template